fastlane/actions/retry_scan_save_failed_tests.rb

```diff
@@ -0,0 +1,157 @@
+require 'nokogiri'
+
+module Fastlane
+  module Actions
+    module SharedValues
+      RETRY_SCAN_SAVE_FAILED_TESTS_CUSTOM_VALUE = :RETRY_SCAN_SAVE_FAILED_TESTS_CUSTOM_VALUE
+    end
+
+    class RetryScanSaveFailedTestsAction < Action
```

Comment:
Wonder if we should move this to the plugin, so it can also be tested... But since I think it will only be used here for now, I think this is ok 👍